app/src/main/java/com/github/h3lp3rs/h3lp/signin/SignInActivity.kt

```diff
@@ -38,14 +39,24 @@ class SignInActivity : AppCompatActivity() {
     /**
      * Check if the current user is already signed in and update activity accordingly
      */
-    private fun checkIfSignedIn() {
-        if (signInClient.isSignedIn()) {
-            userUid = signInClient.getUid()
-            username = getInstance().currentUser?.displayName?.substringBefore(" ")
+    private fun offlineCheckIfSignedIn(){
+        userSignIn = storageOf(Storages.SIGN_IN) // Fetch from storage
```

Comment:
You can import SIGN_IN statically  here too

Reply:
Can't be done unfortunately since it's an enum value :(